README.md

```diff
@@ -134,17 +69,12 @@ Some components need a certain level of interactivity to be usable as React comp
 
 ## Development
 
-`yarn install` and `npm start`. Happy hacking!
-
-You can change the port with the `PORT=` environment variable. Hot reloading
-is setup to reload whenever a file was changed.
-
-For component skeletons, check out the Wiki.
+`yarn install` and `yarn start`. Add or modify stories in `./stories` Happy hacking!
 
 ## Developing while embedded into a react project
 
 `npm link` in this folder. After you changed stuff, run `npm build` to update
```

Comment:
Why not use yarn everywhere? Or is this some stuff that works slightly different with npm?

Reply:
nope, oversight on my part